web/src/main/ui/app/components/flows-new/validators/new-step-dialog.validator.ts

```diff
@@ -3,8 +3,8 @@ import * as _ from "lodash";
 
 export function NewStepDialogValidator(group: FormGroup) {
   let errors = {};
-  if (group.value.type === 'mapping' ||
-      group.value.type === 'mastering') {
+  if (group.value.stepDefinitionType === 'MAPPING' ||
```

Comment:
Please use StepType enum instead of a string

Reply:
Good feedback, thanks.